contrib/terraform/upcloud/cluster-settings.tfvars

```
@@ -152,4 +152,43 @@ server_groups = {
   #   ]
   #   anti_affinity_policy = "yes"
   # }
-}
\ No newline at end of file
+}
+
+connect_router = false
```

Nit: I think that the current name might suggest that you want to connect an existing router, but this option creates a new router and then connects it. I think something like this might be a slightly better description.
```suggestion
enable_router = false
```